site/en/docs/extensions/reference/runtime/index.md

```diff
@@ -2,4 +2,88 @@
 api: runtime
 ---
 
-<!-- Intentionally blank -->
+## Examples
+
+### Use [`getURL`][get-url] to add an extension image to a page {: #example-get-url }
+
+In order for a web page to access an asset hosted on another domain, it must specify the resource's
+full URL (e.g. `<img src-"https://example.com/logo.png">`). The same is true for when a web page
+wants to include assets included in an extension. The two main differences here are that the
+extension's assets must be exposed as [web_accessible_resources][war] and that typically content
+scripts are responsible for injecting extension assets.
+
+This example shows how a [content script][content] can add an image in the extension's package to
+the page that the content script has been [injected][content-inject] into.
+
+```js
+//// content.js
+
+{ // Block used to avoid setting global variables
+  let img = document.createElement('img');
```

Comment:
nit: `const` over `let`

Reply:
Done. I'll follow up with you over chat to discuss const/let conventions more generally.